### ISSUE TITLE
Error parsing time data in aggregate json

### ISSUE THREAD

**Issue by NikolausDemmel** [ternaris/marv-robotics-nodes#7]
I get the following error when rerunning 

After running all nodes once, if I attempt to execute `marv node run` again, I get the following error. It looks like marv chokes on our handcrafted "automated testing" bagfile with a very regular date sometime in 2030.

```
$ marv node run --all-nodes --all-filesets
2016-10-04 09:55:18,893 INFO rospy.topics topicmanager initialized
libdc1394 error: Failed to initialize libdc1394
/marv/venv/local/lib/python2.7/site-packages/matplotlib/font_manager.py:279: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
2016-10-04 09:55:22,938 INFO marv._node Registered nodes in order: ['fileset', u'md5', u'bagset_name', u'bagmeta', u'messages', u'camera_frames', u'diag_count', u'fulltext', u'gnss_plots', u'navsatfix', u'geo_json_trajectory']
2016-10-04 09:56:04,473 WARNING marv._aggregate Exception in aggregate context
Traceback (most recent call last):
  File "/marv/venv/local/lib/python2.7/site-packages/marv/_site.py", line 149, in node_run
    need_run = self.nodes.need_run(agg, rerun=rerun, **kw)
  File "/marv/venv/local/lib/python2.7/site-packages/marv/_node.py", line 178, in need_run
    elif node.name not in agg:
  File "/marv/venv/lib/python2.7/_abcoll.py", line 369, in __contains__
    self[key]
  File "/marv/venv/local/lib/python2.7/site-packages/marv/_aggregate.py", line 185, in __getitem__
    return json_load_agg(f)
  File "/marv/venv/local/lib/python2.7/site-packages/marv/_aggregate.py", line 75, in json_load_agg
    return json.load(*args, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 351, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
  File "/marv/venv/local/lib/python2.7/site-packages/marv/_aggregate.py", line 64, in object_pairs_hook
    return datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f')
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data '2030-03-25T00:00:00' does not match format '%Y-%m-%dT%H:%M:%S.%f'
2016-10-04 09:56:16,381 INFO rospy.core signal_shutdown [atexit]
```

**Comment by NikolausDemmel** [comment]
I can confirm this still happens in 2.0.0b1.

**Comment by NikolausDemmel** [comment]
@chaoflow, this is actually fairly urgent for us, since in the next couple of days (this or next week), I wanted to look into porting our automated testing setup to Marv robotoics, and that currently uses the above "future-dated" bagfile to show the result of jobs aggregating test results over multiple bag files. Currently, it seems that no jobs are successfully run or at least the result not successfully parsed due to the above error.

But in any case I will write you again about our attempt to setup automated testing. Maybe we can avoid this "fake" bagfile all together already with 2.0.0b1.

**Comment by chaoflow** [comment]
should be fixed with https://github.com/ternaris/marv/commit/f7c02c1eb2e39fe1179fc970a5b5d2a39d0608e3 just released as part of 2.0.0b2

**Comment by NikolausDemmel** [comment]
Seems to work. Thanks!
